README.md

```diff
@@ -1,6 +1,15 @@
 ## Book Reading List
 
-Basic python book reading list application. For practicing teamwork and GitHub collaboration. 
+Basic python book reading list application that allows users to store information about books in a database.
+    • Program opens with a main menu that provides choices inlcuding:
+        • Add a book
```

Comment:
In markdown, if these lines start with * then they'll display as a bullet point list. Or if you put a space at the end of each line they'll display one per line. https://guides.github.com/features/mastering-markdown/